Encountify/Services/ScoreboardCreation.cs

```diff
@@ -11,10 +12,9 @@ public class ScoreboardCreation
         public ScoreboardEntry this[int i] => CreateScoreboard().ToArray()[i];
         public List<ScoreboardEntry> CreateScoreboard(bool reversed = false)
         {
-            DatabaseAccess<User> userData = new DatabaseAccess<User>();
+            IUser userData = DependencyService.Get<IUser>();    //new DatabaseAccess<User>();
             List<User> users = (List<User>)userData.GetAllAsync().Result;
-            DatabaseAccess<VisitedLocations> visitedLocationsData = new DatabaseAccess<VisitedLocations>();
-            List<VisitedLocations> visitedLocations = (List<VisitedLocations>)visitedLocationsData.GetAllAsync().Result;
+            DatabaseAccess<VisitedLocations> visitedLocationsData = new DatabaseAccess<VisitedLocations>(); List<VisitedLocations> visitedLocations = (List<VisitedLocations>)visitedLocationsData.GetAllAsync().Result;
```

Comment:
Very long snake.
"Enter" would be nice between those variables LOL

Reply:
Thank you for pointing out, haven't noticed it